test_runner/proguard/config.pro

```diff
@@ -0,0 +1,70 @@
+-dontobfuscate
+-allowaccessmodification
+-keepattributes SourceFile, LineNumberTable
+-dontwarn
+-verbose
+-dontoptimize
+-dontnote kotlin.**
+-dontnote kotlinx.**
+-dontnote com.google.gson.**
+-dontnote proguard.configuration.ConfigurationLogger
+
+-keep class ftl.Main {
+	public static void main(java.lang.String[]);
+}
+
+#Kotlin
+-keepnames class kotlinx.coroutines.internal.MainDispatcherFactory {}
+-keepnames class kotlinx.coroutines.CoroutineExceptionHandler {}
+-keepnames class kotlinx.coroutines.android.AndroidExceptionPreHandler {}
+-keepnames class kotlinx.coroutines.android.AndroidDispatcherFactory {}
+
+-keep class kotlinx.coroutines.internal.MainDispatcherFactory {}
+-keep class kotlinx.coroutines.CoroutineExceptionHandler {}
+-keep class kotlinx.coroutines.android.AndroidExceptionPreHandler {}
+-keep class kotlinx.coroutines.android.AndroidDispatcherFactory {}
+
+# Most of volatile fields are updated with AFU and should not be mangled
+-keepclassmembernames class kotlinx.** {
+    volatile <fields>;
+}
+
+# Proguard configuration for Jackson 2.x
+-keep class com.fasterxml.jackson.databind.ObjectMapper {
+    public <methods>;
+    protected <methods>;
+}
+-keep class com.fasterxml.jackson.databind.ObjectWriter {
+    public ** writeValueAsString(**);
+}
+-keepnames class com.fasterxml.jackson.** { *; }
+-keep class kotlin.Metadata { *; }
+-keep class kotlin.reflect.** { *; }
+
+-keepclassmembers enum * {
+    public static **[] values();
+    public static ** valueOf(java.lang.String);
+}
+
+-keep class com.bugsnag.android.NativeInterface { *; }
+-keep class com.bugsnag.android.Breadcrumb { *; }
+-keep class com.bugsnag.android.Breadcrumbs { *; }
+-keep class com.bugsnag.android.Breadcrumbs$Breadcrumb { *; }
+-keep class com.bugsnag.android.BreadcrumbType { *; }
+-keep class com.bugsnag.android.Severity { *; }
+-keep class com.bugsnag.android.ndk.BugsnagObserver { *; }
+# Picocli
+-keep class picocli.** { *; }
+-keep interface picocli.** { *; }
+
+#ftl
+-keep class ftl.** { *; }
+-keep interface ftl.** { *; }
+
+#google auth
+-keep class com.google.auth.** { *; }
+-keep interface com.google.auth.** { *; }
+
+#ctc wstx
+-keep class com.ctc.wstx.** { *; }
+-keep interface com.ctc.wstx.** { *; }
```

Comment:
```suggestion
-keep interface com.ctc.wstx.** { *; }

```

Reply:
There is an option in Intellij that automatically adds an empty line on save but I don't remember the name.

Reply:
Actually have it installed. It maybe didn't apply it as the file type is not supported